This patch is based on the upstream commit described below, adapted for use in
the Debian package by Peter Michael Green.

commit fd4c315fa371b120eff6145de4feeb3ff0a922e1
Author: Daniel Hofstetter <daniel.hofstetter@42dh.com>
Date:   Thu May 22 16:02:27 2025 +0200

    Bump nix to 0.30.1 and ctrlc to 3.4.7

diff --git a/Cargo.toml b/Cargo.toml
index 35d1edb83..a4c9d3200 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -292,1 +292,1 @@
-ctrlc = { version = "3.4.4", features = ["termination"] }
+ctrlc = { version = "3.4.7", features = ["termination"] }
@@ -314,1 +314,1 @@
-nix = { version = "0.29", default-features = false }
+nix = { version = "0.30", default-features = false }
